### ISSUE TITLE
Add verbose option for output 

### ISSUE THREAD

**Issue by verolero86** [olcf/olcf-test-harness#17]
This is the current output that I think might be good to move (at least majority of it) to a `--verbose` option. This will help minimize the output and most users wouldn't change all this info:
```
Using machine config: frontier.ini
Overriding tasks in inputfile since CLI mode was provided
runmodecmd =  ['start', 'stop']
self.__harness_task:  [['start_tests', None, None]]
self.__harness_task:  [['start_tests', None, None], ['stop_tests', None, None]]
reading harness config /sw/acceptance/olcf-test-harness/configs/frontier.ini
RGT_PATH_TO_SSPACE is already set. Skipping.
App_test_list: [<libraries.apptest.subtest object at 0x7fa807de4748>]
Tasks: [['start_tests', None, None], ['stop_tests', None, None]]
Using machine config: frontier.ini
reading harness config /sw/acceptance/olcf-test-harness/configs/frontier.ini
RGT_MACHINE_NAME is already set. Skipping.
RGT_MACHINE_TYPE is already set. Skipping.
RGT_SCHEDULER_TYPE is already set. Skipping.
RGT_JOBLAUNCHER_TYPE is already set. Skipping.
RGT_CPUS_PER_NODE is already set. Skipping.
RGT_GPUS_PER_NODE is already set. Skipping.
RGT_SUBMIT_QUEUE is already set. Skipping.
RGT_SUBMIT_ARGS is already set. Skipping.
RGT_NCCS_TEST_HARNESS_MODULE is already set. Skipping.
RGT_TYPE_OF_REPOSITORY is already set. Skipping.
RGT_GIT_REPS_BRANCH is already set. Skipping.
RGT_GIT_DATA_TRANSFER_PROTOCOL is already set. Skipping.
RGT_GIT_MACHINE_NAME is already set. Skipping.
RGT_GIT_SERVER_APPLICATION_PARENT_DIR is already set. Skipping.
RGT_GIT_SSH_SERVER_URL is already set. Skipping.
RGT_GIT_HTTPS_SERVER_URL is already set. Skipping.
RGT_PATH_TO_SSPACE is already set. Skipping.
RGT_SYSTEM_LOG_TAG is already set. Skipping.
RGT_ACCT_ID is already set. Skipping.
Generated test unique id: 1636033386.3722255
Creating machine frontier: Type = linux_x86_64 ; Scheduler = slurm ; Job launcher = srun
Creating scheduler
Creating jobLauncher
```

**Comment by hagertnl** [comment]
There is a `--loglevel` option in `runtests.py`, which feeds down to the rgt_logger class. This controls output in the harness launch directory (harness.<datestamp> sub-directory). We could potentially tie this in to the rgt_logger class as well to change the threshold for information logged to the console. For example, all these environment variables would be under the INFO classification, and we could set the default threshold to WARNING, which is a step above INFO. Then the `--verbose` flag could simply change the threshold for console logging back to INFO.

**Comment by hagertnl** [comment]
Should be addressed in #53 , when that issue is resolved.
